molecule/custom-bin-file/playbook.yml

```diff
@@ -0,0 +1,30 @@
+---
+- name: Converge
+  hosts: all
+  vars:
+    - theo_url: https://theo.example.com
```

Comment:
I'm super picky here, pleas double quote values, I've seen nightmares due to colons improperly parsed :D

Reply:
Ok, done